config.ts

```diff
@@ -0,0 +1,51 @@
+const allContent = {
```

Comment:
[Readability] Having everything in a single object is difficult to read, can we have separate objects for each meetup?